src/lib/redux/slices/pitwallSlice/models.ts

```diff
@@ -217,6 +223,34 @@ export interface LiveTiming {
   teamName: string;
 }
 
+export class LiveTimeClass {
```

Comment:
nit pick: can we remove the "Class" suffix from class names?

https://ts.dev/style/#naming-style

Reply:
I have now merged the interface and the class for LiveTiming.